cmd/experimental/kueue-viz/backend/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Build stage
+FROM golang:latest as builder
```

Comment:
use golang 1.23 for consistency (as in the other comment)

Reply:
done